src/osha/oira/services/configure.zcml

```diff
@@ -39,4 +39,13 @@
       name="@navtree"
       />
 
+  <plone:service
+      method="GET"
+      factory=".survey_publication_count.SurveyPublicationCountService"
+      permission="cmf.ManagePortal"
+      for="*"
```

Comment:
This does not match the `survey-publication-count` view:
https://github.com/euphorie/osha.oira/blame/248f3f40023e064897b63606bcf3978bce0ea194/src/osha/oira/content/browser/configure.zcml#L266